capsules/src/crc.rs

```diff
@@ -330,11 +329,7 @@ impl<'a, C: hil::crc::CRC> hil::crc::Client for Crc<'a, C> {
                     }
                     app.waiting = None;
                 })
-                .unwrap_or_else(|err| match err {
-                    Error::OutOfMemory => {}
-                    Error::AddressOutOfBounds => {}
-                    Error::NoSuchApp => {}
-                });
+                .unwrap_or({});
```

Comment:
Why no `err.into()` here?

Reply:
Doesn't work, tried it and was confused for a while.

Reply:
hah, yeah.. turns out that's because the errors were just being ignored.. so.. that's a thing. Probably okay here actually, as the error is just can't report to an app that doesn't exist that the CRC operation is finished. Also fixed in #959.